post/urls.py

```diff
@@ -2,8 +2,12 @@
 from . import views
 
 urlpatterns = [
-   re_path (r'^post/$', views.AddPost.as_view(),name ='add'),
-   re_path (r'^list/', views.PostList.as_view(),name = 'list'),
-   path ('post/<int:pk>', views.PostDetail.as_view(),name = 'detail')
+#    re_path(r'api-auth/', include('rest_framework.urls')), 
+    re_path (r'^list/', views.PostList.as_view(),name = 'list'),
```

Comment:
[trivial]
There are a number of style issues here.  Firstly, there should always be whitespace (space or newline) following a comma.  Secondly, keyword arguments (e.g. `name`) should *not* have whitespace surrounding their `=`.  There should also not be spaces between the function name and parameter list in invocation expressions.

```suggestion
    re_path(r'^list/', views.PostList.as_view(), name='list'),
```